src/content/docs/style-guide/quick-reference/titles.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Titles vs Short titles vs Path vs Redirects
```

Comment:
Style guide: shouldn't only the word "Titles" be initial cap? Also, shouldn't there be a period after each "vs" or will that break the title itself?